docs/contracts/node-operators-registry.md

```diff
@@ -116,46 +95,489 @@ function getOperatorIds()
+
+### setOperatorRewardAddress()
+
+Allows node operator to update reward address
+
+```sol
+function setOperatorRewardAddress(address _rewardAddress)
+        external
+```
+
+#### Parameters:
+
+| Name             | Type      | Description        |
+| ---------------- | --------- | ------------------ |
+| `_rewardAddress` | `address` | New reward address |
+
+## DAO Methods
+:::note
+This methods can be called by DAO-only roles.
+:::
+
+### setDefaultMaxDelegateLimit()
+
+Allows the DAO to set the operator defaultMaxDelegateLimit.
+
+```sol
+function setDefaultMaxDelegateLimit(uint256 _defaultMaxDelegateLimit)
+        external
+        userHasRole(DAO_ROLE)
+```
+#### Parameters:
+
+| Name            | Type      | Description                       |
+| --------------- | --------- | --------------------------------- |
+| `_defaultMaxDelegateLimit`           | `uint256` | Default Max Delegate Limit                 |
+
+
+### setMaxDelegateLimit()
+Allows the DAO to set the operator maxDelegateLimit.
+
+```sol
+function setMaxDelegateLimit(uint256 _operatorId, uint256 _maxDelegateLimit)
+        external
+        userHasRole(DAO_ROLE)
+```
+
+#### Parameters:
+
+| Name                | Type      | Description                              |
+| ------------------- | --------- | ---------------------------------------- |
+| `_operatorId`               | `uint256` | Node Operator id                         |
+| `_maxDelegateLimit` | `uint256`  | Maximum delegate limit |
+
+### setCommissionRate()
+
+Allows the DAO to set the commission rate used.
+
+```sol
+function setCommissionRate(uint256 _commissionRate)
+        external
+        userHasRole(DAO_ROLE)
+```
+#### Parameters:
+
+| Name                | Type      | Description                              |
+| ------------------- | --------- | ---------------------------------------- |
+| `_commissionRate`               | `uint256` | Commission rate                         |
+
+
+### updateOperatorCommissionRate()
+
+Allows the DAO to update commission rate for an operator.
+
+```sol
+function updateOperatorCommissionRate(
+        uint256 _operatorId,
+        uint256 _newCommissionRate
+    ) external userHasRole(DAO_ROLE)
+```
+
+#### Paramet# NodeOperatorsRegistry
+
+- [Source Code](https://github.com/Shard-Labs/PoLido/blob/main/contracts/NodeOperatorRegistry.sol)
+- [Deployed Contract](https://goerli.etherscan.io/address/0xb1f3f45360Cf0A30793e38C18dfefCD0d5136f9a)
+
+Node Operators act as validators on the Goerli chain for the benefit of the protocol. The DAO selects node operators and adds their addresses to the NodeOperatorRegistry contract. Authorized operators have to generate a set of keys for the validation and provide the public key used on heimdall to the smart contract. As Matic is received from users, it is distributed between all active operators. The contract contains a list of operators, their public keys, the logic for managing their state by the DAO, and the .
```

Comment:
Could you improve the last sentence of this paragraph?

Reply:
In general, this part was mistakenly copied from the top of the file. Please remove it here and rewrite it since it is not accurate.

Reply:
I have rewritten it to reflect what I think it should be. Please let me know if you have any suggestions.